pfcli/service/client/data.py

```diff
@@ -166,41 +165,39 @@ def abort_mpu(self, dataset_id: int, path: str, upload_id: str) -> None:
         )
 
     def _multipart_upload_file(
-        self, dataset_id: int, file_path: str, url_dict: Dict[str, Any], ctx: tqdm
+        self,
+        dataset_id: int,
+        file_path: str,
+        url_dict: Dict[str, Any],
+        ctx: tqdm,
+        max_workers: int,
```

Comment:
Maybe we could use a common abstraction for `dataset` and `checkpoint`?

Reply:
I added `UploadableClientService` which defines the interface for uploading files using pre-signed URLs.